client/src/components/signup/signup.js

```diff
@@ -1,55 +1,28 @@
- import React from 'react';
+import React, { Component } from 'react';
 
-// class Signup extends Component {
-//     state = {};
 
-
-
-//     render() {
-//         return (
-//             <div>
-//                 <br />
-//                 <form method = 'POST' action='/register'>
-//                     <input type="text" name="firstName" placeholder="First Name" />
-//                     <br />
-//                     <input type="text" name="lastName" placeholder="Last Name" />
-//                     <br />
-//                     <input type="text" name="password" placeholder="Password" />
-//                     <br />
-//                     <input type="text" name="confirmPassword" placeholder="Confirm Password" />
-//                     <br />
-//                     <input type="email" name="email" placeholder="email" />
-//                     <br />
-//                     <button>Submit</button>
-//                 </form>
-//             </div>
-//         );
-//     }
-// }
-
-// export default Signup;
-
-
-class Signup extends React.Component {
+class Signup extends Component {
+    state = {};
     // constructor() {
     //     super();
-    //     this.handleSubmit = this.handleSubmit.bind(this);
+    // this.handleSubmit = this.handleSubmit.bind(this);
     // }
 
     handleSubmit(event) {
-        event.preventDefault();
-        // const data = new FormData(event.target);
+       // event.preventDefault();
         const firstName = event.target.firstName.value;
         const lastName = event.target.lastName.value
         const email = event.target.email.value
+        const password = event.target.password.value
 
-         console.log(firstName, lastName, email);
+         console.log(firstName, lastName, password, email);
```

Comment:
In general it's best to remove print statements because it's easy to lose track of them and leave too many in your app, making your console messy
Keep this in mind for the future